deployment/terraform/coordinator.go

```diff
@@ -138,19 +165,23 @@ func (t *Terraform) StopCoordinator() error {
 	}
 	ip := output.Agents.Value[0].PublicIP
 
-	extAgent, err := ssh.NewAgent()
+	id := fmt.Sprintf("%s-coordinator-%d", t.config.ClusterName, 0)
+
+	mlog.Info("Stopping the coordinator", mlog.String("ip", ip))
+
+	apiURL := fmt.Sprintf("http://%s:4000/coordinator/%s", ip, id)
+	client := &http.Client{}
```

Comment:
Why not use `http.DefaultClient`? It is effectively the same thing.

Reply:
No reason. That code should be gone as soon as we find the time to create a simple wrapper so didn't spend any time and went with the first thing I thought of.

Reply:
I was actually annoyed that `http` didn't provide a `Delete()` method which meant I had to create the request.